setup.py

```diff
@@ -40,5 +40,5 @@
         'focal_plane': ['scikit-image', 'pandas'],
         'full': ['h5py', 'sqlalchemy', 'pymssql<3.0', 'pandas', 'tables', 'scikit-image', 'healpy', 'pyarrow'],
     },
-    package_data={'GCRCatalogs': ['catalog_configs/*.yaml']},
+    package_data={'GCRCatalogs': ['catalog_configs/*.yaml','site_config/*.yaml']},
```

Comment:
```suggestion
    package_data={'GCRCatalogs': ['catalog_configs/*.yaml', 'site_config/*.yaml']},
```

Reply:
Similar to above.  I guess you want a space following the comma?  I've added it.